Zend/zend_operators.c

```diff
@@ -122,7 +122,7 @@ static _locale_t current_locale = NULL;
 
 #define BLOCKCONV_LOAD(input) \
 	int8x16_t blconv_operand = vld1q_s8((const int8_t*)(input)); \
-	uint8x16_t blconv_mask = vcltq_s8(vaddq_s8(blconv_operand, blconv_offset), blconv_threshold);
+	uint8x16_t blconv_mask = vcltq_s8(vreinterpretq_s8_u8(vaddq_u8(vreinterpretq_u8_s8(blconv_operand), vreinterpretq_u8_s8(blconv_offset))), blconv_threshold);
```

Comment:
`vreinterpretq_u8_s8` converts `uint8x16_t` to `int8x16_t`. `vreinterpretq_s8_u8` is the opposite.

Since `vaddq_s8` overflows, use P`vaddq_u8` to prevent overflow, so need to convert the argument of `vaddq_u8` to `unsigned`. Then convert it to `signed` to pass it to `vcltq_s8`.